moveit_core/CMakeLists.txt

```diff
@@ -114,6 +115,7 @@ catkin_package(
     moveit_distance_field
     moveit_kinematics_metrics
     moveit_dynamics_solver
+    moveit_utils
```

Comment:
this is not a package, remove

Reply:
this section is about what libraries are exported by the moveit_core package

Reply:
oops sorry. the github code folding hid the relevant section and i made a wrong assumption